back/engines/free/email_campaigns/spec/services/email_campaigns/delivery_service_spec.rb

```diff
@@ -76,6 +76,14 @@ def self.consentable_roles
         })
       end
     end
+
+    it 'doesn\'t raise errors while processing all types of campaigns' do
+      service.campaign_classes.each do |klaz|
+        factory_type = :"#{klaz.name.demodulize.underscore}_campaign"
+        create(factory_type)
+      end
+      expect { service.send_on_schedule(Time.now) }.not_to raise_error
+    end
```

Comment:
Nice 'catch-all' test :-)

Reply:
This should hopefully detect bugs like we introduced two weeks ago or so in the future 😅